src/core/entities/MRDivEntity.js

```diff
@@ -119,7 +119,7 @@ export class MRDivEntity extends MREntity {
      * @function
      * @description Callback function of MREntity - connects the background geometry of this item to an actual UIPlane geometry.
```

Comment:
can you add `(async)` to the description like you did for the other newly async functions?